src/lib.rs

```diff
@@ -65,4 +65,4 @@ pub(crate) mod unix_async {
 }
 
 /// This type provides the result type used everywhere in Hyprland-rs
-pub type Result<T> = std::result::Result<T, shared::HyprError>;
+pub type Result<T> = std::result::Result<T, shared::HyprError>;
```

Comment:
Why is this line changed?

Reply:
I don't know what happened with this line. Actually nothing had changed. I have no Idea to how fix it, neither there is no problem with it. as there is nothing had changed in this line I think there is no problem with it.